policyDefinitions/Compute/deploy-windows-vm-app/azurepolicy.json

```diff
@@ -66,8 +66,24 @@
 						"equals": "Microsoft.Compute/virtualMachines"
 					},
 					{
-						"field": "Microsoft.Compute/virtualMachines/storageProfile.osDisk.osType",
-						"equals": "Windows"
+						"anyOf": [
+							{
+								"field": "Microsoft.Compute/virtualMachines/storageProfile.osDisk.osType",
+								"contains", "Windows"
```

Comment:
please update the comma in between contains and windows to be a colon, this was flagged as not valid JSON.
"contains", "Windows" to "contains": "Windows".

Thank you for making this policy definition better!